src/forest/method/h.ts

```diff
@@ -335,6 +352,28 @@ export function h(tag: string, opts?: any) {
           }
         })
       })
+      draft.classList.forEach(property => {
+        // We know if `name` is store, `enabled` also is store
+        if (is.unit(property.name) || is.unit(property.enabled)) {
+          draft.seq.push({
+            type: 'classList',
+            field: is.unit(property.name)
+              ? property.name
+              : createStore(property.name),
+            value: is.unit(property.enabled)
+              ? property.enabled
+              : createStore(property.enabled),
+          })
+          if (is.unit(property.name)) processStoreRef(property.name)
+          if (is.unit(property.enabled)) processStoreRef(property.enabled)
```

Comment:
```suggestion
          const $name = is.unit(property.name)
              ? property.name
              : createStore(property.name)
          const $enabled = is.unit(property.enabled)
              ? property.enabled
              : createStore(property.enabled)
          
          draft.seq.push({
            type: 'classList',
            field: $name,
            value: $enabled,
          })
          processStoreRef($name)
          processStoreRef($enabled)
```
Not sure that this is important, but let's proceed each created store